packages/design-system/tests/setupTests.js

```diff
@@ -1,8 +1,9 @@
 import { configure } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
```

Comment:
I know it's not an official one, but it's still in progress: https://github.com/enzymejs/enzyme/pull/2430
However `@wojtekmaj` is one of the contributors to the official package, so I decided to go with his solution.